ui/src/ElasticSearch.elm

```diff
@@ -83,6 +83,48 @@ extractClauses kind field values =
                     ]
 
 
+prepareSearchQuery : String -> String
+prepareSearchQuery search =
+    search
+        |> String.split " "
+        |> List.map (\w -> "*" ++ w ++ "*")
+        |> String.join " "
+
+
+buildSearchClause : String -> Maybe EncodedFilter
+buildSearchClause search =
+    case search of
+        "" ->
+            Nothing
+
+        search ->
+            Just <|
+                Encode.object
+                    [ ( "query_string"
+                      , Encode.object
+                            [ ( "query", Encode.string <| prepareSearchQuery search )
+                            , ( "analyzer", Encode.string "standard" )
+                            , ( "default_operator", Encode.string "AND" )
+                            , ( "phrase_slop", Encode.int 1 )
+                            , ( "auto_generate_phrase_queries", Encode.bool True )
+                            , ( "analyze_wildcard", Encode.bool True )
+                            , ( "lenient", Encode.bool True )
+                            , ( "split_on_whitespace", Encode.bool True )
+                            , ( "fields"
+                              , [ "source.product"
+                                , "target.channel^1.2"
+                                , "target.version^10"
+                                , "target.locale^3"
+                                , "target.platform^2"
```

Comment:
are you limiting the fields on purpose (instead of searching on `_all`)? And if so, would it make sense to have it explained somewhere (eg the placeholder)?